metricbeat/module/redis/info/info.go

```diff
@@ -1,4 +1,114 @@
-// Loads data from the redis info command
+/**
+
+This calls the info command in redis and retrieves the data.
+
+
+The document sent to elasticsearch has the following structure:
+
+	{
+	  "metricset": "info",
+	  "module": "redis",
+	  "redis-info": {
+	    "clients": {
+	      "blocked_clients": "0",
+	      "client_biggest_input_buf": "0",
+	      "client_longest_output_list": "0",
+	      "connected_clients": "3"
+	    },
+	    "cluster": {
+	      "cluster_enabled": "0"
+	    },
+	    "cpu": {
+	      "used_cpu_sys": "210.63",
+	      "used_cpu_sys_children": "0.00",
+	      "used_cpu_user": "113.11",
+	      "used_cpu_user_children": "0.00"
```

Comment:
Doubles as strings in JSON could be a problem, _if_ explicit ES mappings are not set. As I think explicit mapping are needed anyway, I think it's good to keep it like this, otherwise we'd have the types defined in two places.